core/src/lib/auth/internal/CoreStitchAuth.ts

```diff
@@ -137,28 +141,23 @@ export default abstract class CoreStitchAuth<TStitchUser extends CoreStitchUser>
    *
    * - returns: An `any` representing the decoded response body.
    */
-  public doAuthenticatedJSONRequest(
-    stitchReq: StitchAuthDocRequest
-  ): Promise<any> {
-    return this.doAuthenticatedJSONRequestRaw(stitchReq)
-      .then(response => EJSON.parse(response.body, {strict: false}))
-      .catch(err => { throw StitchError.wrapDecodingError(err) })
-  }
+  public doAuthenticatedJSONRequest<T>(
+    stitchReq: StitchAuthDocRequest,
```

Comment:
[minor] AFAIK now that we properly encode bodies at request build time there's no more reason for this to be a `StitchAuthDocRequest`, it can just be a `StitchAuthRequest`, and you can rename it to `doAuthenticatedRequest<T>`